bluebrain/repo-bluebrain/packages/py-neurodamus/package.py

```diff
@@ -12,6 +12,7 @@ class PyNeurodamus(PythonPackage):
     git = "https://github.com/BlueBrain/neurodamus.git"
 
     version("develop", branch="main")
+    version("2.17.1", commit="493e8ea")
```

Comment:
`nitpick`: if this is based on a commit, it should be added as `alpha` or something similar?

Reply:
I am wondering. I see neuron uses `a[number]` but py-libsonata uses normal versions. What's the rule?

Reply:
Spack handles alpha versions very poorly. Don't do that :)

@WeinaJi `2.17.0.1` would work, or adding the date, like `2.17.0-20240122`